ts/src/connections/tunnelHostBase.ts

```diff
@@ -53,14 +55,20 @@ export class TunnelHostBase
      */
     public hostPrivateKeyPromise?: Promise<KeyPair>;
 
+    /**
```

Comment:
Should we move `connectionProtocol` property to the base class? This way both host and client can use it.

Reply:
Done.